apps/app.mento.org/.env.example

```
@@ -11,3 +11,12 @@ NEXT_PUBLIC_ENABLE_DEBUG="true"
 # Banner configuration (optional)
 NEXT_PUBLIC_BANNER_TEXT=
 NEXT_PUBLIC_BANNER_LINK=
+
+# Trove-history subgraph (borrow activity panel). Optional — @repo/web3 falls
+# back to the currently-deployed Subgraph Studio URLs when these are unset.
+# Set them after a subgraph redeploy in mento-protocol/bold changes the Studio
+# version label, or to move mainnet onto the decentralized-network gateway.
+NEXT_PUBLIC_TROVES_SUBGRAPH_URL=
+NEXT_PUBLIC_TROVES_SUBGRAPH_URL_CELO_SEPOLIA=
+# Required only when a troves subgraph URL above points at gateway.thegraph.com.
+NEXT_PUBLIC_GRAPH_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GRAPH_API_KEY key should go before the NEXT_PUBLIC_TROVES_SUBGRAPH_URL key

(UnorderedKey)